src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,102 @@
+package duke;
+
+public class Ui {
+
+    /**
+     * Constructor to create a UI object.
+     */
+    Ui() {}
+
+    private final static String LOGO = " ____        _        \n"
+                                    + "|  _ \\ _   _| | _____ \n"
+                                    + "| | | | | | | |/ / _ \\\n"
+                                    + "| |_| | |_| |   <  __/\n"
+                                    + "|____/ \\__,_|_|\\_\\___|\n";
+
+    private final static String HORIZONTAL_DIVIDE =
+        "___________________________________________________________________";
+
+    /**
+     * Prints out a welcome message.
+     */
+    public void welcomeUser() {
+        System.out.println("Hello from\n" + LOGO);
+        System.out.println("What can Duke do for you today?");
+        System.out.println(HORIZONTAL_DIVIDE);
+    }
+
+    /**
+     * Prints out the added entry.
+     *
+     * @param entry THe newly-added entry.
+     * @param id The #id of the newly-added entry.
+     */
+    public void addEntry (Entry entry, int id) {
```

Comment:
Seems like you don't need a white space between the method name and the parentheses.